pipelines/jobs/configurations/jdk17.groovy

```diff
@@ -61,4 +61,12 @@ triggerSchedule_nightly="TZ=UTC\n30 03 * * 3,5"
 // 12:05 Sun
 triggerSchedule_weekly="TZ=UTC\n05 12 * * 7"
 
+// scmReferences to use for weekly release build
+weekly_release_scmReferences=[
```

Comment:
Can we add a note to `RELEASING.md` that this variable will need to be adjusted when JDK head is bumped?

Reply:
Do you mean reset back to "" HEAD ?

Reply:
added